package/gluon-mesh-vpn-wireguard/check_site.conf

```diff
@@ -0,0 +1,17 @@
+local function check_peer(k)
```

Comment:
This shoud actually be `check_site.lua` not `.conf`.

Reply:
wow. of course.